Frameworks/include/BridgeHelpers.h

```diff
@@ -132,7 +132,7 @@ static inline bool shouldUseConcreteClass(Class self, Class base, Class derived)
 
 // Helper macro for implementing allocWithZone
 #define ALLOC_PROTOTYPE_SUBCLASS_WITH_ZONE(NSBridgedType, NSBridgedPrototypeType)                            \
-    (NSObject*) allocWithZone : (NSZone*)zone {                                                              \
+    (NSObject*)allocWithZone : (NSZone*)zone {                                                               \
```

Comment:
separated out commits are wonderful though 

---
In reply to: [113529420](https://github.com/Microsoft/WinObjC/pull/2538#discussion_r113529420) [](ancestors = 113529420)